.gitlab-ci.yml

```diff
@@ -0,0 +1,41 @@
+test-installer:
+  stage: test
+  image: centos:8
```

Comment:
Can we run this against Centos Streams instead?  I think Andrew has the necessary bits to convert a Centos 8 to Streams.  Seems wise since that is the future...

Reply:
> I'm curious how other tests would look and what actually causes these tests to be run.
Fair point. I'll include at least a placeholder in the next commit.

Reply:
> Can we run this against Centos Streams instead? I think Andrew has the necessary bits to convert a Centos 8 to Streams. Seems wise since that is the future...
I believe 'centos-stream' is the image, need to test.